src/lib/shapes/star.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { BaseShape } from './base';
+
+type StarProps = {
+  size: string | string[];
+  color: string;
+  top?: string | string[];
+  left?: string | string[];
+  right?: string | string[];
+  bottom?: string | string[];
+  className?: string;
+  zIndex?: string;
+  breakpoints?: number[];
+  position?: string;
+};
+
+export const Star = ({
+  size,
+  color,
+  top = ``,
+  position = `absolute`,
+  left = ``,
+  right = ``,
+  bottom = ``,
+  className = `anim-shape-star`,
+  zIndex = `-1`,
+  breakpoints = [600, 900, 1200]
+}: StarProps) => {
+  const StyledStar = styled('div')`
+    position: inherit;
+    height: inherit;
+    width: inherit;
+    background: ${color};
+    top: inherit;
+    clip-path: polygon(
```

Comment:
create a seperate file `starPath.tsx` which will contain `clip-path` for this shape. We don't want to include constants in the same file.